src/impl_bytemuck.rs

```diff
@@ -97,25 +151,38 @@ unsafe impl bytemuck::checked::CheckedBitPattern for Mix {
 
 #[cfg(test)]
 mod tests {
-    use crate::{Compose, Extend, Fill, Mix};
+    use crate::{Compose, Extend, Fill, ImageFormat, ImageQuality, Mix};
     use bytemuck::{checked::try_from_bytes, Contiguous, Zeroable};
     use core::ptr;
 
     #[test]
     fn checked_bit_pattern() {
-        let valid = bytemuck::bytes_of(&1_u8);
+        let valid_zero = bytemuck::bytes_of(&0_u8);
+        let valid_one = bytemuck::bytes_of(&1_u8);
```

Comment:
This is just `&[0u8]`/`&[1u8]`, right? Just want to check my understanding of what is happening here (to be clear, the form in this PR is fine; for consistency with other tests)